src/index.html

```diff
@@ -210,95 +254,163 @@
+      </div>
+      <textarea id="customConfig" rows="5" class="w-full"></textarea>
+      <div class="mt-2">
+        <input type="checkbox" id="clearStorageCheckbox" />
+        <label for="clearStorageCheckbox">Clear saved viewer state before applying</label>
+      </div>
+      <div class="mt-2">
+        <button id="setConfigButton" class="controls-button">Apply Configurations</button>
+        <button id="resetConfigButton" class="controls-button">Reset to Default Configuration</button>
+      </div>
+    </div>
 
-        <div class="controls-group">
-          <h2 class="controls-title">Configuration builder</h2>
-          <div id="config-builder">
-          </div>
-        </div>
-        
+    <!-- Configuration Builder -->
+    <div class="controls-group collapsible-section">
+      <button class="collapsible-heading collapsible-icon caret-down"><strong>Configuration builder</strong></button>
+      <div class="collapsible-content" style="display: none;">
+        <p>Use the dropdown menus to customise the Universal Viewer. Your selections will automatically populate the custom configuration box above. You will have to click the Apply Configurations button when you are ready to put your changes into effect.</p>
+      </div>
+        <div id="config-builder"></div>
       </div>
+    </div>
+    <script>
+      document.querySelectorAll('.collapsible-heading').forEach(button => {
+        button.addEventListener('click', () => {
+        const isOpen = button.classList.contains('caret-up');
+        const content = button.nextElementSibling;
+
+        if (isOpen) {
+          button.classList.remove('caret-up');
+          button.classList.add('caret-down');
+          content.style.display = 'none';
+        } else {
+          button.classList.remove('caret-down');
+          button.classList.add('caret-up');
+          content.style.display = 'block';
+        }
+      });
+    });
+    </script>
+
+  <script>
+  function openTab(evt, tabName) {
+    document.addEventListener("DOMContentLoaded", function () {
+    document.querySelector(".tablinks").click(); 
+});
+
+    const tabcontent = document.getElementsByClassName("tabcontent");
+    const tablinks = document.getElementsByClassName("tablinks");
+    for (let i = 0; i < tabcontent.length; i++) {
+      tabcontent[i].style.display = "none";
+    }
+    for (let i = 0; i < tablinks.length; i++) {
+      tablinks[i].classList.remove("active");
+    }
+
+    document.getElementById(tabName).style.display = "block";
+
+    evt.currentTarget.classList.add("active");
+    document.addEventListener("DOMContentLoaded", function () {
+    document.querySelector(".tablinks").click();  
+});
+
+  }

```

Comment:
More indentation issues:

```suggestion
  function openTab(evt, tabName) {
    document.addEventListener("DOMContentLoaded", function () {
      document.querySelector(".tablinks").click();
    });

    const tabcontent = document.getElementsByClassName("tabcontent");
    const tablinks = document.getElementsByClassName("tablinks");
    for (let i = 0; i < tabcontent.length; i++) {
      tabcontent[i].style.display = "none";
    }
    for (let i = 0; i < tablinks.length; i++) {
      tablinks[i].classList.remove("active");
    }

    document.getElementById(tabName).style.display = "block";

    evt.currentTarget.classList.add("active");
    document.addEventListener("DOMContentLoaded", function () {
      document.querySelector(".tablinks").click();  
    });
  }
```